### ISSUE TITLE
[ENHANCEMENT] Temporary new files

### ISSUE THREAD

**Issue by yashpalgoyal1304** [Zettlr/Zettlr#2023]
## Description

"Temporary" files within zettlr

## Proposed Changes

- A new option for "Do not prompt for directory when creating new files" to complement "Do not prompt for filenames when creating new files".
- So the predefined directory should be such that it remains unaffected by:
    - closing app, 
    - shutting down pc, 
    - app updates, 
    - if possible then even unaffectd from window updates too i guess.
- Like they are in notepad++. 
- This type of directory is like data/profile directory in some apps if i recall correct.

Result:
That is, I just click new file and start typing, and exit when done. The next time i open, i see all the files (the whole session) right there.

They were zettlr v1.2.3 too i guess, although i am not sure if the temporary files in that version were saved for next session.

## Caveats

i dont think so.

## Additional Information

i dont think, but feel free to ask

**Comment by nathanlesage** [comment]
What we have now is the ability to create new files without having to save them first. However, these are not persisted across programm shutdowns. The idea itself makes sense but I cannot promise whether that will make it into 2.0 immediately.

**Comment by yashpalgoyal1304** [comment]
> What we have now is the ability to create new files without having to save them first

oh wow, how? even that would suffice for the time being.

**Comment by nathanlesage** [comment]
Yeah, since I'm also one power-user of Zettlr, I recognise inefficient things myself pretty quickly, and I was finally fed up with having to always create files before starting to write! But buffering those unsaved files on disk is pretty hard, and there are too many other construction sites, so that's more the icing on the cake that can wait ;)

**Comment by nathanlesage** [comment]
Thanks to stalebot I stumbled upon this issue: These things have been implemented into the app already.

**Comment by yashpalgoyal1304** [comment]
hi, which commit/mr etc. that'd may be helpful later :)

**Comment by nathanlesage** [comment]
I don't know, but it's available in the nightlies: https://nightly.zettlr.com
